openshift/cronJob-template.yaml

```diff
@@ -23,7 +23,7 @@ objects:
       app: thoth-core
   spec:
     schedule: '*/30 * * * *'
-    suspend: true
+    suspend: ${THOTH_SUSPEND_JOB}
```

Comment:
do we need some " around here?!

Reply:
I don't think so - I think it can be directly passed here.

Reply:
It has to be without quotes otherwise I get an error "unrecognized type: bool"